src/components/MainPage/ProductPagination/ProductPagination.js

```diff
@@ -0,0 +1,124 @@
+import { useEffect, useMemo, useState } from "react"
+import { useDispatch, useSelector } from "react-redux"
+import { useHistory } from "react-router"
+import './ProductPagination.css'
+import * as ActionCreators from '../../../store/cart'
+
+const ProductPagination = ({ productList = [], sort = 0, filters = [], ...props }) => {
+    const [currPage, setCurrPage] = useState(1)
+    const [is4PerRow, setIs4PerRow] = useState(false)
+    const [searchQuery, setSearchQuery] = useState('')
+    const cart = useSelector(state => state.cart)
+    const perRow = is4PerRow ? 4 : 3
```

Comment:
consts instead of numbers

Reply:
perRow3=3,perRow4=4?
נראה קצת מיותר למקרה הזה